index.php

```diff
@@ -59,11 +59,18 @@
 
 	$save_path = "{$FROTZ_SAVE_PATH}/{$session_id}.zsav";
 
-	#
+
+
+
+	handle_save_and_restore($command, $save_path, $session_id, $FROTZ_SLOT_SAVES_ENABLED, $FROTZ_SAVE_SLOTS, $FROTZ_SAVE_PATH);
```

Comment:
The one issue nagging at me is the fact that this function could just stop the script (by calling `exit()` inside) for a non-error case and it's not obvious by the calling function that this could happen.
I'm thinking `handle_save_and_restore` should return `true`/`false` depending on if it's handled the command or not. And the caller of `handle_save_and_restore` will make the call to `exit()`. What this will give us is a clearer understanding when viewing the function call that it can indeed `exit()` if a save or restore call is handled.

It should still `exit()` on error though.

This is just an idea, and I'd like your thought.